src/cli/Runtime/ExtConfig.cs

```diff
@@ -15,25 +14,39 @@ public class ExtConfig
         public ExtConfig()
         {
             Linters = new List<ExtLint>();
+
         }
 
         public class ExtLint
         {
             public ExtLint()
             {
+                Config = new Config();
             }
 
             [JsonProperty("name")]
             public string Name { get; set; }
 
             [JsonProperty("config")]
-            public object Config { get; set; }
+            public Config Config { get; set; }
```

Comment:
Are you sure that it should be `Config` and not `object`? How to store custom configuration for linter?